ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleInterstitialAdapter.java

```diff
@@ -302,71 +312,18 @@ protected void onDetachedFromWindow() {
     if (mBannerRequest == null) {
       // Adapter does not support multiple Banner instances playing for same placement except for
       // Refresh
-      mMediationBannerListener.onAdFailedToLoad(
+      mediationBannerListener.onAdFailedToLoad(
           VungleInterstitialAdapter.this, AdRequest.ERROR_CODE_INVALID_REQUEST);
       return;
     }
 
     mBannerRequest.setAdLayout(adLayout);
-    mBannerRequest.setVungleListener(mVungleBannerListener);
 
     Log.d(TAG, "Requesting banner with ad size: " + adConfig.getAdSize());
```

Comment:
```suggestion
    Log.d(TAG, "Requesting banner with ad size: " + adConfig.getAdSize());
    this.mediationBannerListener = mediationBannerListener;
```

Reply:
@HuFangshuai is this somehow related to https://github.com/Vungle/googleads-mobile-android-mediation/pull/75/files#r496916265?
mediationBannerListener should be available from VungleBannerAdapter until VungleInterstitialAdapter was not GCed, and I believe no event will be delivered to the publisher after that anyway

Reply:
Hi @alexander-gubov , I tested base on PR https://github.com/googleads/googleads-mobile-android-mediation/pull/271 and found the click event not called due to mediationBannerListener is null, but actually the banner is playing correctly.

Reply:
@HuFangshuai hmmm how often did the mediationBannerListener get GC'd?

Reply:
@HuFangshuai @jiancchen 
They have removed strong ref to MediationBannerListener, that's why it is GCted and event were missing on that branch:
https://github.com/googleads/googleads-mobile-android-mediation/pull/271/files#diff-e3eb809fec019433eb4442f5941b9e9fL228